doozer/doozerlib/backend/rebaser.py

```diff
@@ -428,6 +428,26 @@ class MergeSourceResult:
             source_dockerfile_content = source_dockerfile.read()
             distgit_dockerfile.write(source_dockerfile_content)
 
+        gomod_path = dest_dir.joinpath('go.mod')
+        if gomod_path.exists():
+            # Read the gomod contents
+            new_lines = []
+            with open(gomod_path, "r") as file:
+                lines = file.readlines()
+                for line in lines:
+                    stripped_line = line.strip()
+                    match = re.match(r"(^go \d\.\d+$)", stripped_line)
```

Comment:
I'd permit whitespace+ before and after the number, but unlikely to be important.

Reply:
thanks - when I checked, I didn't see a case like that. If I do, will update the regex